Source/DafnyDriver/Commands/TestCommand.cs

```diff
@@ -50,4 +50,6 @@ static class TestCommand {
     });
     return result;
   }
+
+  public static void EnsureStaticConstructorHasRun() { }
```

Comment:
Wait, is it possible the static constructor to not run if you don't call this method?

Reply:
If you don't use a class, then it's static constructor is not run.